packages/cli/test/fixtures/unit/vercel-env-rm/.gitignore

```diff
@@ -0,0 +1,3 @@
+.next
```

Comment:
Not a huge deal, but organizationally, fixtures can be placed into `test/fixtures/unit/commands/<name>` if they are specific to a subcommand.